components/Login/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Form, Input, Button, Checkbox, Typography } from "antd";
+import { MailOutlined, LockOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import Link from "next/link";
+
+import styles from "./style.module.css";
+
+function LoginForm() {
+  const { Title, Text } = Typography;
+  const { errors, isLoading, login } = useAuth();
+  // TODO: Session must be remembered when 'remember' is true
+  const onFinish = ({ email, password /*, remember*/ }) => {
+    login({ email, password });
+  };
+
+  return (
+    <Form
+      name="login"
+      className={styles["login-form"]}
```

Comment:
Since CSS modules are self-contained, and we don't really need to write classes that long, we could simplify and use only `.login` which allows to access it as `styles.login` and have a little more help from the linter and autocomplete editor feature.